src/network/networkprotocol.h

```diff
@@ -987,12 +987,12 @@ enum NetProtoCompressionMode {
 };
 
 const static std::string accessDeniedStrings[SERVER_ACCESSDENIED_MAX] = {
-	"Invalid password",
+	"The password is invalid.",
 	"Your client sent something the server didn't expect.  Try reconnecting or updating your client",
 	"The server is running in simple singleplayer mode.  You cannot connect.",
 	"Your client's version is not supported.\nPlease contact server administrator.",
-	"Player name contains disallowed characters.",
-	"Player name not allowed.",
+	"This player name contains disallowed characters.  Allowed are a-z, A-Z, 0-9, hyphen and underscore.",
+	"This player name is reserved and can not be used.  Please use a different player name.",
```

Comment:
Why add "Please use a different player name." to one reason, but not to the other?

Reply:
No need for 'this'.
'Allowed are a-z, A-Z, 0-9, hyphen and underscore' is good and useful.
'Please use a different player name.' is obvious and not needed.